test/integration/corechecks/docker/serviceup_test.go

```diff
@@ -1,18 +0,0 @@
-// Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
-// This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-present Datadog, Inc.
-
-package docker
-
-import (
-	"testing"
-
-	"github.com/DataDog/datadog-agent/pkg/metrics/servicecheck"
-)
-
-func TestServiceUp(t *testing.T) {
-	expectedTags := []string{}
-
-	sender.AssertServiceCheck(t, "docker.service_up", servicecheck.ServiceCheckOK, "", expectedTags, "")
```

Comment:
Do the service check related tests need to be migrated? I see there is a unit test with this case https://github.com/DataDog/datadog-agent/blob/bb7f635482751a8929903ec5795b8dead2098a5e/pkg/collector/corechecks/containers/docker/check_test.go#L242 but wasn't sure if there are any for the exitcode tests